app/models/mail_handler.rb

```diff
@@ -161,8 +161,8 @@ def receive_issue
     end
 
     issue = WorkPackage.new(author: user, project: project)
-    issue.safe_attributes = issue_attributes_from_keywords(issue)
-    issue.safe_attributes = { 'custom_field_values' => custom_field_values_from_keywords(issue) }
+    issue.attributes = issue_attributes_from_keywords(issue)
```

Comment:
Yep, I paused there for a minute but specs reported no failures, so I assumed the change is ok. Maybe we can add a test case for statues?

For `issue_attributes_from_keywords`, it looks fine to me. We are constructing the hash ourselves (so essentially what strong_params would allow).